integration-test/snapshots/explain/federation_integration_tests__tpch_tpch_q4_explain.snap

```diff
@@ -3,20 +3,20 @@ source: integration-test/src/main.rs
+|               |         TableScan: orders projection=[o_orderkey, o_orderdate, o_orderpriority], full_filters=[orders.o_orderdate >= Utf8("1993-07-01")]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| physical_plan | SchemaCastScanExec                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|               |   VirtualExecutionPlan name=duckdb compute_context=tpch_sf1.db initial_sql=SELECT "orders"."o_orderpriority", count(1) AS "order_count" FROM "orders" WHERE (("orders"."o_orderdate" < (CAST('1993-07-01' AS DATE) + INTERVAL '3 MONS')) AND EXISTS (SELECT "lineitem"."l_orderkey", "lineitem"."l_partkey", "lineitem"."l_suppkey", "lineitem"."l_linenumber", "lineitem"."l_quantity", "lineitem"."l_extendedprice", "lineitem"."l_discount", "lineitem"."l_tax", "lineitem"."l_returnflag", "lineitem"."l_linestatus", "lineitem"."l_shipdate", "lineitem"."l_commitdate", "lineitem"."l_receiptdate", "lineitem"."l_shipinstruct", "lineitem"."l_shipmode", "lineitem"."l_comment" FROM "lineitem" WHERE ("lineitem"."l_orderkey" = "orders"."o_orderkey") AND ("lineitem"."l_commitdate" < "lineitem"."l_receiptdate"))) AND ("orders"."o_orderdate" >= '1993-07-01') GROUP BY "orders"."o_orderpriority" ORDER BY "orders"."o_orderpriority" ASC NULLS LAST |
```

Comment:
Learning purpose - does `initial_sql` is now already rewrittent_sql or just naming matches remote tables? I see there are multiple `rewritten_xxx` steps possible but I don't see them in snapshots.

Reply:
The initial_sql is just what the initial SQL looks like after converting the initial logical plan before any rewrites. The other rewritten steps aren't emitted if there is no difference. I will add a step to emit the final sql that will be sent to the execution engine.

Reply:
I'm not sure I understand, this is handling the rewrite

Reply:
Ah, I understand now - the name registered in datafusion and the name in DuckDB are the same, so this isn't exercising the rewrite path properly.